app/api/resources/mentorship_relation.py

```diff
@@ -232,6 +233,61 @@ def put(cls, request_id):
 
         return response
 
+@mentorship_relation_ns.route("mentorship_relation/<int:request_id>/task/<int:task_id>/comment/<int:comment_id>/report")
+class ReportViolationForComment(Resource):
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc("report_violation_for_comment")
+    @mentorship_relation_ns.expect(auth_header_parser)
+    @mentorship_relation_ns.response(
+        HTTPStatus.OK, "%s" % messages.VIOLATION_WAS_REPORTED_SUCCESSFULLY
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.FORBIDDEN, "%s" % messages.USER_CANT_REPORT_THEIR_OWN_COMMENT
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.BAD_REQUEST, "%s" % messages.UNACCEPTED_STATE_RELATION
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.UNAUTHORIZED,
+        "%s\n%s\n%s\n%s"
+        % (
+            messages.TOKEN_HAS_EXPIRED,
+            messages.TOKEN_IS_INVALID,
+            messages.AUTHORISATION_TOKEN_IS_MISSING,
+            messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION,
+        ),
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.NOT_FOUND,
+        "%s\n%s\n%s\n%s"
+        % (
+            messages.MENTORSHIP_RELATION_DOES_NOT_EXIST,
+            messages.TASK_DOES_NOT_EXIST,
+            messages.TASK_COMMENT_DOES_NOT_EXIST,
+            messages.TASK_COMMENT_WITH_GIVEN_TASK_ID_DOES_NOT_EXIST,
+        ),
+    )
+    def put(cls, request_id, task_id, comment_id):
```

Comment:
This should be a POST because this feature is about creating a new record of CoC violations.

Reply:
Done

Reply:
done